easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart

```diff
@@ -30,85 +33,98 @@ class NFTsListTile extends StatelessWidget {
     bottomSheet.show();
   }
 
+  void openOwnerView() {
+    Navigator.of(navigatorKey.currentState!.context).pushNamed(
+      RouteUtil.kOwnerViewScreen,
+      arguments: publishedNFT,
+    );
+  }
+
   Widget getPublishedCard({required BuildContext context}) {
-    return DecoratedBox(
-      decoration: BoxDecoration(
-        color: EaselAppTheme.kWhite,
-        boxShadow: [
-          BoxShadow(
-            color: Colors.black.withOpacity(0.3),
-            offset: const Offset(0.0, 1.0),
-            blurRadius: 4.0,
-          ),
-        ],
-      ),
-      child: Padding(
-        padding: EdgeInsets.symmetric(horizontal: 15.w, vertical: 15.h),
-        child: Row(
-          children: [
-            SizedBox(
-              height: 45.h,
-              width: 45.w,
-              child: NftTypeBuilder(
-                onImage: (context) => buildCachedNetworkImage(publishedNFT.url.changeDomain()),
-                onVideo: (context) => buildCachedNetworkImage(publishedNFT.thumbnailUrl.changeDomain()),
-                onPdf: (context) => buildCachedNetworkImage(publishedNFT.thumbnailUrl.changeDomain()),
-                onAudio: (context) => buildCachedNetworkImage(publishedNFT.thumbnailUrl.changeDomain()),
-                on3D: (context) => ModelViewer(
-                  src: publishedNFT.url.changeDomain(),
-                  backgroundColor: EaselAppTheme.kWhite,
-                  ar: false,
-                  autoRotate: false,
-                  cameraControls: false,
-                ),
-                assetType: publishedNFT.assetType.toAssetTypeEnum(),
-              ),
-            ),
-            SizedBox(
-              width: 10.w,
+    return InkWell(
+      key: const Key(kNftListTileKey),
+      onTap: () {
+        context.read<CreatorHubViewModel>().openOwnerView(openOwnerViewPressed: openOwnerView);
```

Comment:
Why not use `openOwnerView()`

Reply:
We need to write test case for this that's why need to implement this function in viewModel. Modified `openOwnerView` function